src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/TicketTabNav_PurchaseTabScreen.tsx

```diff
@@ -72,25 +71,24 @@ export const TicketTabNav_PurchaseTabScreen = ({navigation}: Props) => {
         });
         return;
       }
-    } else {
-      if (
-        fareProductTypeConfig.configuration.requiresTokenOnMobile &&
-        !hasInspectableMobileToken
-      ) {
-        navigation.navigate(
-          'Root_ActiveTokenOnPhoneRequiredForFareProductScreen',
-          {
-            nextScreen: {
-              screen: 'Root_PurchaseOverviewScreen',
-              params: {
-                fareProductTypeConfig,
-                mode: 'Ticket',
-              },
+    } else if (
+      fareProductTypeConfig.configuration.requiresTokenOnMobile &&
+      !hasInspectableMobileToken &&
+      !isFallbackState
```

Comment:
Should it also check other MobileTokenStatus states here?

Reply:
Not completely sure about this. I saw it is now added an additional check regarding if mobile token status is `error`, but in that case it means that fallback isn't enabled. 

@jorelosorio At least add this situation to the acceptance criteria so it is up in the light, and notify Sebastian about the acceptance criteria change, to see if he agrees.